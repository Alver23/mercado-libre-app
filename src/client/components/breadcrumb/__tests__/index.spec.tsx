// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import Breadcrumb from '../index';

describe('<Breadcrumb />', () => {
  let component: ShallowWrapper;
  const items = ['item 1', 'Ã¬tem 2'];

  beforeEach(() => {
    component = shallow(<Breadcrumb items={items} />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
    expect(component.find('p').text()).toBe(items.join(' > '));
  });
});
