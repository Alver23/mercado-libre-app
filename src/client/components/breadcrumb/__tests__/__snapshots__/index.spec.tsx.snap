// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> should render correctly 1`] = `
<div
  className="my-4"
>
  <p
    className="text-xs md:text-sm text-gray-500"
  >
    item 1 &gt; Ã¬tem 2
  </p>
</div>
`;
